src/System.Runtime.WindowsRuntime/src/System/IO/WindowsRuntimeStreamExtensions.CoreRT.cs

```diff
@@ -0,0 +1,361 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
Remove this file --- CoreCLR and CoreRT use same wrapper instead. see new PR https://github.com/dotnet/corefx/pull/12354